.github/workflows/dispatch.yml

```diff
@@ -0,0 +1,42 @@
+# Trigger the execution of copy-workflow.yml, in batches of 50 repositories.
+# This workflow is needed since GitHub Actions limits the matrix size to 256 jobs.
+# We use one job per repository per file.
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ closed ] # IMPORTANT: check the merge status for every job: github.event.pull_request.merged == true
+
+env:
+  MAX_REPOS_PER_WORKFLOW: 50
```

Comment:
I think we should reduce this number to something smaller than the number of repos we currently have in our `config.json`, just to make sure that we end up with multiple batches to exercise the mechanism defined here?

Reply:
Reduced to 5.